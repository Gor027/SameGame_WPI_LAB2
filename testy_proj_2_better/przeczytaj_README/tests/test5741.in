 @@@BB@ B @    B BB@B@@B BBB@  BBBB @  BB B@B B@B@@ @BB B@B B B@B@ B@ @@B@@ @
 B @B@@ B  @ @ @@@B@ @@B@@ B  @ @@@B @BB@@B B@  B@@ B @B@@ @ 
@@ @   @@@@     @ BB  B@@B@@ @  @ B BB@ B@B@B@  @BB @B@@@@BB@B@B@ @B@@BB@BB  @ 
B@ BB  B@@ B@@ BB@B B@@BB B
B@B@ BBB@B B  @@  @B@ @@ @B@ BBB BBB @  B B B@ @@    @ @@B@ @ @B@B@ B   B@B @B@@
    B@  @BBB   @@ BBB@@B   B@  @BB@ B BBB@@@ @@@B B@@ B@B @     @B B@B B
 B@  @BB @B@ @  B@@@ @@@ B@ B @BB@@@@BB B @@ BB@B BB  B B B BB@  @@@BB@@
 B@@@@B   BBB  @
BB@ B  
 BBB  @  B@@ B  @B @ @BB@B@   @ @BB @   @  @BBB@ 
