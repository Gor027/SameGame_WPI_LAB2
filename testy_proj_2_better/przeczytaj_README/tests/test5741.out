                      @B
           @      B@  @B          B     B
   @ B@ B@@B   B@ B@ @B@ BB     BB@     @ B  @  B       B
 @@@B@@ B@@B   @@BBB @@B @@B    @@B @   B B@B@ @B @   B @ B         @
 BB@BBB@@BBB @ BB@BBB@@@@BBB  @B@BB B@B B B@B@B@@ @ @ @B@@B B B B@ B@  @
 @@BB@BB@BB@ @ @@@@BB@@B@@B@@ BBBB@BBB@BB@B@B@@@@@B BB@@@@@ @ @@@B@BB@@BB@@
 @@@B@BB@@BB@@ @B@B@@@@@BB@@@@@@B@@B@@BBB@@@BBB@B@B@@BB@B@@@@B@B@B@B@BBB@BB@@
@@@@@@@B@B@@BB@@@B@@B@BB@B@BBB@B@BB@@BBB@@@@BBB@BBBB@@B@BBBBBB@B@@@@BB@@B@B@B@@
